packages/engineer/src/cli.ts

```diff
@@ -42,6 +42,7 @@ program
     .option('--title <title>', 'application title to display in browser')
     .option('--publicConfigsRoute <publicConfigsRoute>', 'public route for configurations')
     .option('--engineerEntry <engineerEntry>', 'entry feature for engineer', 'engineer/gui')
+    .option('--featureDiscoveryRoot <featureDiscoveryRoot>', 'package subdirectory where feature discovery starts')
```

Comment:
Needs to have a default value ('.') and also add to build script below

Reply:
Added, thanks.